workbench-core/infrastructure/src/workbenchCognito.ts

```diff
@@ -87,13 +80,15 @@ export interface WorkbenchUserPoolOidcIdentityProvider
 
 export class WorkbenchCognito extends Construct {
   public readonly userPool: UserPool;
-  public readonly userPoolClient: UserPoolClient;
+  public readonly webUiUserPoolClient: UserPoolClient;
+  public readonly integrationTestUserPoolClient: UserPoolClient;
```

Comment:
Not necessarily for integration tests. Maybe something more generic like "programmaticAccessUserPoolClient"